src/saftey_preset.h

```diff
@@ -0,0 +1,113 @@
+#pragma once
+
+#include "types.h"
+#include "core/extension.h"
+
+namespace librealsense
+{
+	// Convenience blocks
+	//struct float2 { float x, y; };
+	//struct float3 { float x, y, z; };
+	//struct float3x3 { float3 x, y, z; };  // column-major
+#pragma pack(push, 1)
+
+	struct safety_extrinsics_table
+	{
+		float3x3            rotation;           // Rotation matrix Sensor->Robot CS (""Leveled World"" assumed)
+		float3              translation;        // Metric units
+	};
+
+	enum class safety_zone_type : uint8_t
+	{
+		e_zone_danger = 0,

```

Comment:
The only way to reference the enum class is through `safety_zone_type ::`
so I am not sure we need the `e_zone` prefix
we can use : 
```
danger = 0,
warning = 1,
...
```
directly